pyvistaqt/counter.py

```diff
@@ -1,17 +1,14 @@
-"""
-This module contains a basic Qt-compatible counter class.
-"""
-from PyQt5.QtCore import QObject, pyqtSignal, pyqtSlot
+"""This module contains a basic Qt-compatible counter class."""
+
+from qtpy.QtCore import QObject, Signal, Slot
```

Comment:
This will immediately break any `pyvistaqt` code without deprecation. We colud make a small compatibility layer just by making a little `._qtpy` file that contains:
```
try:
    from qtpy import QtCore
except ImportError:
    warn('qtpy not found; falling back to PyQt5. qtpy will be required in pyvistaqt 0.4')
    from PyQt5.QtCore import ...
else:
    from qtpy import ...
```
And then in this file do:
```suggestion
from ._qtpy.QtCore import QObject, Signal, Slot
```
And once we cut version 0.3, we can revert the `from ._qtpy import ...` lines to just be `from qtpy import ...`

But maybe this is overkill since `pyvistaqt` is still relatively new. Up to the main `pyvista` devs I guess...

Reply:
If the core developers think there needs to be a deprecation period, then this would be a good solution. I'm open to this.

Reply:
I think just going ahead without the depreciation works.  Users can always downgrade to an older version, and adding in a depreciation is a bit of a pain.  Plus, installing `qtpy` should be in the `setup.py`, so it will be included upon install/upgrade.

We can always give a more descriptive error on import if we wish, but it's a bit of extra work making it backwards compatible and adding a depreciation warning as well.